dev/e2e/pubspec.yaml

```diff
@@ -0,0 +1,11 @@
+name: e2e
+description: Flutter end-to-end test.
+
+dependencies:
+  flutter:
+    sdk: flutter
+dev_dependencies:
```

Comment:
Added `package:test` dep to `dev_dependencies`.
Moved `package:flutter_driver` dep from `dev_dependencies` to `dependencies`.